shell/src/test/java/org/apache/accumulo/shell/ShellConfigTest.java

```diff
@@ -126,7 +126,7 @@ public void testZooKeeperHostFallBackToSite() throws Exception {
     Map<String,String> data = new HashMap<>();
     data.put(Property.INSTANCE_ZK_HOST.getKey(), "site_hostname");
     AccumuloConfiguration conf = new ConfigurationCopy(data);
```

Comment:
Seems like this configuration is being created and not used.  Does this test pass?